unlock-protocol-com/next.config.js

```diff
@@ -46,17 +46,16 @@ Object.keys(requiredConfigVariables).forEach((configVariableName) => {
     )
   }
 })
-module.exports = {
+
+const nextConfig = {
   publicRuntimeConfig: requiredConfigVariables,
   webpack(config) {
     config.module.rules.push({
       test: /\.md$/,
       use: 'raw-loader',
     })
-    config.module.rules.push({
```

Comment:
Let's move to another PR? It looks like a signigicant config change that I don't want to break :)

Reply:
Okay. Though it's not used anymore.